### PR TITLE
add empty list state

### PR DESCRIPTION
**Pull request by ngoldman**
screen shot 2016-03-07 at 3 27 05 pm

### CODE REVIEW

File: renderer/index.css [hunk 1]
```diff
@@ -253,7 +253,20 @@ body.drag::before {
...
+  opacity: .5;
+  padding: 24px;
+  margin: 6px;
+  text-align: center;
+  border: 5px #444 dashed;
+  flex: 1 1 auto;
+}
+
+.lead {
+  font-size: 150%;
```
Comment (feross) [inline renderer/index.css]: We've been using `em` throughout.

Comment (ngoldman) [inline renderer/index.css]: ah alright, I'll change that

**Comment by feross** [comment]
YES! I was about to add this.

File: renderer/index.css (hunk 1 above)
Comment (ngoldman) [inline renderer/index.css]: actually I'm not even using it, feel free to delete it if you get to it first.

**Pull request merged**